README.md

```diff
@@ -35,6 +36,7 @@ The sanctioned addresses can be extracted with this tool from the
 [`sdn_advanced.xml`][2] file. The tools supports the following output formats:
 - `TXT` file format (one address per line)
 - `JSON` file containing a list of addresses
+- `CSV` file containing a list of addresses
```

Comment:
Isn't a CSV with one column the same as a TXT file with one address per line?

Reply:
usually non-IT users familiar the csv format